### PR TITLE
grammatical correction

### PR DESCRIPTION
**Pull request by njmsaikat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 t-digest
 ========
 
-A new data structure for accurate on-line accumulation of rank-based statistics such as quantiles
+A new data structure for accurate online accumulation of rank-based statistics such as quantiles
```
Comment (tdunning) [inline README.md:4]: this is a matter of opinion. Different folks prefer different ways.

Happy to move to your form

File: README.md [hunk 2]
```diff
@@ -18,8 +18,8 @@ compact, particularly when serialized.
 In summary, the particularly interesting characteristics of the t-digest are that it
 
 * has smaller summaries when serialized
-* works on double precision floating point as well as integers.
-* provides part per million accuracy for extreme quantiles and typically <1000 ppm accuracy for middle quantiles
+* works on double-precision floating points as well as integers.
```
Comment (tdunning) [inline README.md:21]: This usage of floating point as a noun by analogy with integers is interesting. My feeling, however, is that floating point here is an adjective, not a noun (an elision of "floating point number", contrast with "integer number") and thus should not be made plural.

File: README.md [hunk 3]
```diff
@@ -18,8 +18,8 @@ compact, particularly when serialized.
...
+* provides part per million accuracies for extreme quantiles and typically <1000 ppm accuracy for middle quantiles
```
Comment (tdunning) [inline README.md:22]: This should not be made plural.

File: README.md [hunk 4]
```diff
@@ -32,7 +32,7 @@ Recent News
 There is a [new article (open access!)](https://www.sciencedirect.com/science/article/pii/S2665963820300403) in Software Impacts on 
 the t-digest, focussed particularly on this reference implementation.
  
-Lots has happened in t-digest lately. Most recently, with the help of people
+Lots have happened in t-digest lately. Most recently, with the help of people
```
Comment (tdunning) [inline README.md:35]: The standard idiom in American English is "lots has happened". Let's stick with that.

File: README.md [hunk 5]
```diff
@@ -41,7 +41,7 @@ continuing. As part of introducing this change to the sort, I made the core
 implementation pickier about enforcing the size invariants which forced updates
 to a number of tests.
 
-The basic gist of other recent changes are that the core algorithms have been 
+The basic gist of other recent changes is that the core algorithms have been
```
Comment (tdunning) [inline README.md:44]: Good catch on this one. There didn't use to be more than one change!

File: README.md [hunk 6]
```diff
@@ -143,9 +143,9 @@ artifacts.
 The major areas are
  
  * core - this is where the t-digest and unit tests live
- * docs - the main paper and auxillary proofs live here
+ * docs - the main paper and auxiliary proofs live here
```
Comment (tdunning) [inline README.md:146]: Thank you for catching this.

Comment (tdunning) [inline README.md:146]: Thank you for catching this.

File: README.md [hunk 7]
```diff
@@ -143,9 +143,9 @@ artifacts.
...
  * benchmarks - this is the code that tests the speed of the digest algos
- * quality - this is the code that generates and analyzes accuracy information
+ * quality - this is the code that generates and analyzes accuracy of information
```
Comment (tdunning) [inline README.md:148]: I don't agree with this. the code analyzes information about accuracy. The information is the output of the test, not hte information contained in the t-digest.

File: README.md (hunk 6 above)
Comment (tdunning) [inline README.md:146]: Thank you for catching this.

File: README.md [hunk 8]
```diff
@@ -85,7 +85,7 @@ sizes near the tails. We now have 4 scale functions supported for both
 major digest forms (`MergingDigest` and `AVLTreeDigest`) to allow
 different trade-offs in terms of accuracy.
 
-These scale functions now have associated proofs that they all
+These scale functions now have associated proof that they all
```
Comment (tdunning) [inline README.md:88]: I think that there are multiple proofs (one each for each scale function)

What is the rationale for the singular here? I may be off base.

**Review by tdunning — CHANGES_REQUESTED** [review]
I think that I agree with about 2/3 of these. Some are obvious improvements (like correting the speling of auxillary), but I strongly disagree some.

Can we come to consensus on these?

**Comment by njmsaikat** [comment]
made requested changes https://github.com/tdunning/t-digest/pull/187

**Pull request closed without merging**